src/services/realtime_monitor.py

```
@@ -220,58 +220,108 @@ def add_data_spec(self, data_spec: str) -> bool:
+        background thread.
         """
-        importer = DataImporter(database=self.database, batch_size=self.batch_size)
         fetcher = RealtimeFetcher(sid=self.sid)
-        logger.info("Sequential monitoring loop started", specs=self.data_specs)
+        jvlink = fetcher.jvlink
+        updater = RealtimeUpdater(database=self.database)
+
+        # Initialize JV-Link once for the session
+        try:
+            jvlink.jv_init()
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)